siphon/tests/test_catalog_access.py

```diff
@@ -82,6 +82,22 @@ def test_dataset_download(nids_url):
         os.remove(temp)
 
 
+@recorder.use_cassette('cat_to_open')
+def test_dataset_default_download(nids_url):
```

Comment:
Only wondering if we should avoid managing the temporary directory manually and instead use the pytest [`tmpdir`](https://docs.pytest.org/en/latest/tmpdir.html) fixture.